compiler/modules/front.syntax/token.bal

```diff
@@ -499,6 +528,57 @@ public readonly class SourceFile {
     }
 }
 
+function tokenEndCodePointIndex(string[] fragments, FragCode[] fragCodes, int startCodePointIndex) returns int {
+    int fragmentIndex = fragmentCountUpTo(fragments, startCodePointIndex);
+    return lineTokenEndIndex(fragments, fragCodes, startCodePointIndex, fragmentIndex);
+}
+
+function fragmentCountUpTo(string[] fragments, int startCodePointIndex) returns int {
```

Comment:
Let's call the second argument `codePointIndex`.

Reply:
fixed